### PR TITLE
ESC-864 - added in nudge email endpoint to backend

### PR DESCRIPTION
**Pull request by WitchellJack**

### CODE REVIEW

File: app/uk/gov/hmrc/eusubsidycompliance/models/EmailRequest.scala [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ */
+
+package uk.gov.hmrc.eusubsidycompliance.models
+
+import play.api.libs.json.{Json, OFormat}
+
+case class EmailRequest(
+  undertakingIdentifier: String,
+  businessEntityIdentifier: String,
+  messageType: String,
```
Comment (habeeb01) [inline app/uk/gov/hmrc/eusubsidycompliance/models/EmailRequest.scala:24]: is `messageType` a String in their schema? I think it should be an Integer

File: app/uk/gov/hmrc/eusubsidycompliance/syntax/FutureSyntax.scala [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import scala.concurrent.Future
+
+object FutureSyntax {
```
Comment (habeeb01) [inline app/uk/gov/hmrc/eusubsidycompliance/syntax/FutureSyntax.scala:36]: do we need this?

File: conf/app.routes [hunk 3]
```diff
@@ -15,3 +15,5 @@ POST       /subsidy/update                                  uk.gov.hmrc.eusubsid
 POST       /undertaking/disable                             uk.gov.hmrc.eusubsidycompliance.controllers.UndertakingController.disableUndertaking()
 
 POST       /subsidy/retrieve                                uk.gov.hmrc.eusubsidycompliance.controllers.UndertakingController.retrieveSubsidies()
+
+POST       /send-nudge-email                                uk.gov.hmrc.eusubsidycompliance.controllers.EmailController.sendNudgeEmail()
```
Comment (habeeb01) [inline conf/app.routes:19]: I think the expected url is /email-notification. Please confirm, I think its mentioned in the specs

File: conf/application.conf [hunk 4]
```diff
@@ -93,6 +93,11 @@ mongodb {
   uri = "mongodb://localhost:27017/eu-subsidy-compliance"
 }
 
+email {
```
Comment (habeeb01) [inline conf/application.conf:96]: might be good to add something to make it obvious that these are templateIds. Maybe rename it or add a comment

File: conf/app.routes (hunk 3 above)
Comment (WitchellJack) [reply conf/app.routes:19]: Have checked this, correct it is /email-notification good spot!

File: conf/application.conf (hunk 4 above)
Comment (WitchellJack) [reply conf/application.conf:96]: I agree, have added a comment

File: app/uk/gov/hmrc/eusubsidycompliance/models/EmailRequest.scala (hunk 1 above)
Comment (WitchellJack) [reply app/uk/gov/hmrc/eusubsidycompliance/models/EmailRequest.scala:24]: Having checked the docs it seems to be a String! I thought it should of been an Int at first too, but Alas :D

**Review by habeeb01 — APPROVED** [review]
LGTM 👍

**Review by habeeb01 — APPROVED** [review]

**Pull request merged**